### ISSUE TITLE
Failing encryption, all keys except mine

### ISSUE THREAD

**Issue by IS2511** [open-keychain/open-keychain#2403]
## Expected Behavior

Encrypted text in copyboard

## Current Behavior

```
[START] Начало операции подписи/шифровки
    [WARN] Некорректный ключ для шифрования: 0xb7efbf94ea1809d1
```
^ Incorrect encryption key ^
The key worked perfectly for like a year. I think that's an app error.

## Steps to Reproduce (for bugs)

1. Add a public key
2. Try to encrypt some text(/files) -> fail
3. Update the key from the key server
4. Try to encrypt some text(/files) -> fail
5. Try to encrypt with any key -> fail
6. Try to encrypt with my public key -> success

## Context

Trying to encrypt some text

## Your Environment

* Android Version: 8.0.0 8.0.0.372(C185) EMUI 8.0.0
* Device Model: Huawei MHA-L29
* OpenKeychain Version: 5.3-beta1 (53000)
* From Google Play or F-Droid?: GP

**Comment by dschuermann** [comment]
Confirmed. This is a bug in our current beta version.

**Comment by dschuermann** [comment]
fixed in 5.3-beta2
